python/openvino_tokenizers/utils.py

```diff
@@ -11,6 +11,8 @@
 from openvino import Model, Type
 from openvino.preprocess import PrePostProcessor
 from openvino.runtime import opset12 as opset
+from openvino.runtime import Model, PartialShape, Type
+from openvino.runtime.utils.types import make_constant_node
```

Comment:
```suggestion
from openvino import Model, PartialShape, Type
from openvino.utils.types import make_constant_node
```
Could you check if this works?

Reply:
Outdated. Since i reverted adding statefulness this diff is not present anymore.